lib/common/util.js

```diff
@@ -80,6 +80,43 @@ exports.readYamlFileAsync = function (filePath) {
 
 
 /**
+ * Takes the given directory path and file and replace tags found in the file with
+ *   values from the tag list
+ * 
+ * @param {Array} listTag - An array of objects containing fields regex: {regex} and value: {String}
+ * @param {String} filePath - The full path to the file on disk to do substitutions to
+ * @param {String} fileName - The name of the file on disk
+ * @returns {Promise.<String>} - A Promise of a String representing the contents of fileName with substitutions
+ */
+exports.replaceTagInFile = function (listTag, filePath, fileName) {
+  return new Promise((resolv,reject)=>
```

Comment:
Could we change this to use readFileSync and writeFileSync? I know those are blocking calls, but Handel doesn't need to worry too much about that since we don't have multiple users being serviced in a web service, and the predominant time spent in the library is waiting for AWS resources to create. I like readFileSync and writeFile sync because we can reduce this function to a much more readable few lines rather than using callbacks and promises.